domain_test.go

```diff
@@ -28,49 +28,48 @@ func TestTemplating(t *testing.T) {
 	})
 
 	assert.NoError(t, err)
-	assert.Equal(t, `<domain type='kvm'>
+	assert.Equal(t, `<domain type="kvm">
   <name>domain</name>
-  <memory unit='MB'>4096</memory>
-  <vcpu placement='static'>4</vcpu>
-  <features><acpi/><apic/><pae/></features>
-  <cpu mode='host-passthrough'>
-    <feature policy="disable" name="rdrand"/>
-  </cpu>
+  <memory unit="MB">4096</memory>
+  <vcpu placement="static">4</vcpu>
   <os>
-    <type arch='x86_64'>hvm</type>
-    <boot dev='hd'/>
-    <bootmenu enable='no'/>
+    <type arch="x86_64">hvm</type>
+    <boot dev="hd"></boot>
+    <bootmenu enable="no"></bootmenu>
   </os>
   <features>
-    <acpi/>
-    <apic/>
-    <pae/>
+    <pae></pae>
+    <acpi></acpi>
+    <apic></apic>
   </features>
-  <clock offset='utc'/>
+  <cpu mode="host-passthrough">
+    <feature policy="disable" name="rdrand"></feature>
+  </cpu>
+  <clock offset="utc"></clock>
   <on_poweroff>destroy</on_poweroff>
   <on_reboot>restart</on_reboot>
   <on_crash>destroy</on_crash>
   <devices>
-    <disk type='file' device='disk'>
-      <driver name='qemu' type='qcow2' cache='default' io='threads' />
-      <source file='machines/domain/domain.test'/>
-      <target dev='vda' bus='virtio'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="qcow2" cache="default" io="threads"></driver>
+      <source file="machines/domain/domain.test"></source>
+      <target dev="vda" bus="virtio"></target>
     </disk>
-    <graphics type='vnc' autoport='yes' listen='127.0.0.1'>
-      <listen type='address' address='127.0.0.1'/>
-    </graphics>
-    <console type='pty'></console>
-    <channel type='pty'>
-      <target type='virtio' name='org.qemu.guest_agent.0'/>
+    <interface type="network">
+      <mac address="52:fd:fc:07:21:82"></mac>
+      <source network="network"></source>
+      <model type="virtio"></model>
+    </interface>
+    <console></console>
```

Comment:
the `type='pty'` is gone, dunno if this matters?

Reply:
yes, apparently it's by default. When I use this driver, create the VM, then run by hand `virsh dumpxml`, `type='pty'` is present.

Reply:
Yup, indeed. Can be tested relatively easily with
```
$ virsh -c qemu:///system dumpxml crc >crc.xml
$ virsh -c test:///default
virsh # define crc.xml
virsh # dumpxml crc